README.md

```diff
@@ -1,29 +1,92 @@
-    - If you have a better example feel free to use it, but always aim for a real world example. And you can use that example throughout the rest of your reading and/or use other new examples. 
-    * The answer is tuples and this sets the stage for writing the content that covers your learning objectives above.--- You can iterate over the key-value pairs in a dictionary with a for-in loop. Each item in the dictionary is returned as a (key, value) tuple. Explain how you can decompose the tuple’s members into temporary constants as part of the iteration.
+```swift
+
+func startGame() {
+    let dice: Int = throwDice()
+    print("Congrats - you rolled \(dice)!")
+}
+
+//simulate  dice throw
+func throwDice() -> Int {
+    print("..calculating random results from die throw..")
     
+    //generate and return a random number - let's just return 5 for now
+    return 5 
+}
+
+startGame()
+
 ```
+
+
+Even though we accurately applied what we know, we’ve encountered some issues.  Our requirements state the game involves the throwing of `two` dice, but our `throwDice` function only allows for `one` dice to be thrown at a time. While we could call the function twice to generate `two` random numbers, it would be ideal if we could somehow call `throwDice` once and have it  return `two` independent values.
+ 
+
+Upon further examination, the `throwDice` return value of `Int` also could cause issues related to readibility. For example, a future developer may wonder if the `Int` return value is indeed the result of the simulated dice throw, or the result of some other calculation.
+
+
+Tuples as Types
+---------------
+Fortunately, `tuples` can be used to solve both issues! Let's rewrite (also called *refactor* in the parlance of software engineers) our game to support this new concept:
+
+
+```swift
+
+func startGame() {
+    
+    //define tuple to simulate dice
+    let dice: (first: Int, second: Int) = throwDice()
+    print("Congrats - you rolled a \(dice.first) and \(dice.second) ! ")
+}
+
+
+func throwDice() -> (first: Int, second: Int) {
+    print("..calculating random results from die throw..")
+
+    //generate and return a random number - let's just return 5 and 1 for now
+    return (5, 1)
+}
+
+startGame()
+
+```
+
+ 
+Here's our game refactored to support tuples. If you've haven't noticed, the `dice` variable included in the `startGame()` function has been updated with an interesting new syntax. Instead of the function returning a single `Int` value, the new type `(first: Int, second: Int)` groups multiple values into a single compound value.
+
+As seen in previous modules, we can declare native Swift types (i.e.`Int`, `String`) as constants, variables or return types. In the case of our refactored `dice` tuple, Swift treats this custom type as value `(first: Int, second: Int)`. In addition, even though `dice` supports two `Int` values, tuples can be of any type and do not have to be of the same type as each other.
+
```

Comment:
> tuples can be of any type and do not have to be of the same type as each other.
After stating this, I like the idea of then showing them what it is you're referring to. A tuple that maybe includes a Bool, Int and String (possibly sticking with your narrative of the dice rolling game).